package.json

```diff
@@ -1,31 +1,6 @@
 {
   "name": "coursetable-root",
-  "private": true,
```

Comment:
?

Reply:
ya fax idk was auto

Reply:
oh it just moved it down

Reply:
Yeah I'm not sure why the keys got reordered